flask_app.py

```diff
@@ -0,0 +1,148 @@
+"""
+I made a simple flask app that creates a SQLAlchemy database that stores simple information like the email, password and token.
+
+To make it work locally you have to install quite some modules.
```

Comment:
Please focus on only adding meaningful comments to the codebase. Anything else can be written as a comment or a Discord message.
```suggestion
```